src/wrapper/jnienv.rs

```diff
@@ -591,6 +591,7 @@ impl<'a> JNIEnv<'a> {
 
     /// Get the class for an object.
     pub fn get_object_class(&self, obj: JObject) -> Result<JClass<'a>> {
+        non_null!(obj, "get_object_class");
         Ok(jni_non_null_call!(self.internal, GetObjectClass, obj.into_inner()))
```

Comment:
This doesn't seem to require exception check (it [does not throw](https://docs.oracle.com/javase/10/docs/specs/jni/functions.html#getobjectclass)).

Reply:
Yes, and I reflected this in the spreadsheet. Do you want me to correct this within this PR?

Reply:
Yes, please.